pkg/apis/azureprovider/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -34,6 +34,11 @@ func (in *AzureClusterProviderSpec) DeepCopyInto(out *AzureClusterProviderSpec)
 	in.EtcdCAKeyPair.DeepCopyInto(&out.EtcdCAKeyPair)
 	in.FrontProxyCAKeyPair.DeepCopyInto(&out.FrontProxyCAKeyPair)
 	in.SAKeyPair.DeepCopyInto(&out.SAKeyPair)
+	if in.DiscoveryHashes != nil {
```

Comment:
This is more of a question? Is this how we deepcopy a string slice in Go?

Reply:
Interesting code, this is part of auto generation, interesting that it generates that piece of code :)

Reply:
Yep, the deepcopy is generated, so no need to review that.